### DISCUSSION TITLE
How to add HUD on a Tilemap ?

### DISCUSSION THREAD

**Discussion by lerouxrgd** [joshuajbouw/bevy_chunk_tiles#179] [Q&A] [answered]
Hello,

I am trying to add some heads up display info on a tilemap, basically something like this:
https://github.com/bevyengine/bevy/blob/v0.5.0/examples/ui/text.rs#L26

However I am under the impression that some special rendering setup is required, would you have any pointer about that?
(For now it looks like the text is rendered just before the tilemap in the first frame, and then never rendered again)

**Comment by lerouxrgd** [comment, marked as answer]
Oh I realized that I just need to put a higher z coordinate to the text for it to be displayed on top of the tilemap...

**Reply by joshuajbouw** [reply]
Cool :D
